I run `./gradlew aggregatedJavadocg` with JDK 11 and this patch and it still produces following error message.
```
> Task :aggregatedJavadoc FAILED
/home/chia7712/kafka/streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java:44: error: cannot find symbol
import org.apache.kafka.streams.internals.KeyValueStoreFacade;
                                         ^
  symbol:   class KeyValueStoreFacade
  location: package org.apache.kafka.streams.internals
/home/chia7712/kafka/streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java:45: error: cannot find symbol
import org.apache.kafka.streams.internals.WindowStoreFacade;
                                         ^
  symbol:   class WindowStoreFacade
  location: package org.apache.kafka.streams.internals
/home/chia7712/kafka/streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java:69: error: cannot find symbol
import org.apache.kafka.streams.processor.internals.TestDriverProducer;
                                                   ^
  symbol:   class TestDriverProducer
  location: package org.apache.kafka.streams.processor.internals
/home/chia7712/kafka/streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java:226: error: cannot find symbol
    private final TestDriverProducer testDriverProducer;
                  ^
  symbol:   class TestDriverProducer
  location: class TopologyTestDriver
4 errors

```

BTW, there is a issue related to this error (https://issues.apache.org/jira/browse/KAFKA-12435)

--- a/build.gradle
+++ b/build.gradle
@@ -59,6 +59,7 @@ allprojects {
   apply plugin: 'idea'
   apply plugin: 'org.owasp.dependencycheck'
   apply plugin: 'com.github.ben-manes.versions'
+  apply plugin: 'java'
 
   dependencyUpdates {
     revision="release"
@@ -166,11 +167,6 @@ if (file('.git').exists()) {
     ])
   }
 }
-
-// Apply 'java' to the root project so that the root-level task 'aggregatedJavadoc' can depend on compileJava
-// Weirdly, this seems to be required for some people, but not others. It doesn't seem to be harmful to add it
-// here, in any case.
-apply plugin: 'java'
 
 subprojects {
 

--- a/build.gradle
+++ b/build.gradle
@@ -167,6 +167,10 @@ if (file('.git').exists()) {
   }
 }
 
+// Apply 'java' to the root project so that the root-level task 'aggregatedJavadoc' can depend on compileJava
+// Weirdly, this seems to be required for some people, but not others. It doesn't seem to be harmful to add it
+// here, in any case.
+apply plugin: 'java'
 
 subprojects {
 
@@ -2278,8 +2282,6 @@ project(':connect:mirror-client') {
     dependsOn copyDependantLibs
   }
 }
-
-apply plugin: 'java'
 
 task aggregatedJavadoc(type: Javadoc, dependsOn: compileJava) {
   def projectsWithJavadoc = subprojects.findAll { it.javadoc.enabled }

--- a/build.gradle
+++ b/build.gradle
@@ -177,7 +177,6 @@ subprojects {
   // eg: ./gradlew allDepInsight --configuration runtime --dependency com.fasterxml.jackson.core:jackson-databind
   task allDepInsight(type: DependencyInsightReportTask) doLast {}
 
-  apply plugin: 'java'
   // apply the eclipse plugin only to subprojects that hold code. 'connect' is just a folder.
   if (!project.name.equals('connect')) {
     apply plugin: 'eclipse'

